customer_service.go

```diff
@@ -591,7 +596,8 @@ func (s *CustomerService) Update(ctx context.Context, identity string, p Custome
 }
 
 // CustomerRemoveParams parameters
-type CustomerRemoveParams map[string]interface{}
+type CustomerRemoveParams struct {
```

Comment:
@vdandugc do we need this change ? since It is a blank struct

Reply:
ya, i think this makes all inputs consistent (to always send as struct), so it certainly makes sense. i will test this api to check it works, good shout.